src/WorkOS.net/Services/UserManagement/UserManagementService.cs

```diff
@@ -108,17 +108,17 @@ public UserManagementService(WorkOSClient client)
         /// <param name="options"> Parameters used to authenticate user with a code.</param>
         /// <param name="cancellationToken">An optional token to cancel the request.</param>
         /// <returns> A User and Session record.</returns>
-        public async Task<(User, Session)> AuthenticateUserWithCode(
+        public async Task<User> AuthenticateUserWithCode(
             AuthenticateUserWithCodeOptions options,
             CancellationToken cancellationToken = default)
         {
             var request = new WorkOSRequest
             {
                 Options = options,
                 Method = HttpMethod.Post,
-                Path = $"/users/sessions/token",
+                Path = $"/users/authenticate",
             };
-            return await this.Client.MakeAPIRequest<(User, Session)>(request, cancellationToken);
+            return await this.Client.MakeAPIRequest<User>(request, cancellationToken);
```

Comment:
The response from the API nests the user under the `user` key, so we'll need to account for that when parsing the response.